scripts/generate-runner-parameters-documentation.py

```diff
@@ -21,12 +21,42 @@
 
 import os
 
-from st2common.bootstrap.runnersregistrar import RUNNER_TYPES
+import yaml
+
+from st2common.content.loader import RunnersLoader, RUNNER_MANIFEST_FILE_NAME
+
 
 CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
 HEADER = '.. NOTE: This file has been generated automatically, don\'t manually edit it'
 
 
+def _get_runners_dir():
+    runner_loader = RunnersLoader()
+    runner_dir = os.path.join(CURRENT_DIR, '../st2/contrib/runners')
+
+    runner_types = runner_loader.get_runners([runner_dir])
+
+    return runner_types
+
+
+def _get_runners():
```

Comment:
In theory you could also move this method `get_runners_from_dir` (or similar) into bootstrap module and re-use it, but I'm also fine with only having it here for now.